tests/testthat/test_xml_diff.R

```diff
@@ -5,38 +5,38 @@ context("XML diff")
 basefile <- 'test-data/modeltime.xml'
 
 test_that('Python is available', {
-  skip_on_os('windows')
+  skip_on_os(c('windows', 'linux'))
```

Comment:
I'm not sure we want to skip this test on ALL Linux machines. What about
```
skip_if(Sys.getenv("SYSTEM_NAME") == "constance")
```
?

Reply:
@bpbond okay will make that change!